lisa/mixin_modules.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+# The file imports all the mix-in types that can be initialized
+# using reflection.
```

Comment:
Can you update the error message in subclasses.py, if the subclass is not found? The error message should be more actionable, either add the imports here, or check the extensions paths.

Reply:
The exception also shows up if you make a spelling mistake in the runbook YAML. So, it possible should not refer to internal details.

Reply:
Yes, we can provide multiple checks what we can.